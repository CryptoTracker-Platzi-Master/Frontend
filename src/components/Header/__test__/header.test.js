import React from "react";
import { BrowserRouter } from "react-router-dom";

import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { Header } from "../";

describe("[ component / Header ]", () => {
  describe("when  `Header` is mounted", () => {
    it("[`Header`]", () => {
      const { asFragment } = render(
        <BrowserRouter>
          <Header />
        </BrowserRouter>
      );
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
