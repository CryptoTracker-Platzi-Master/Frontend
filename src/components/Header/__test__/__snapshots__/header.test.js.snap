// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ component / Header ] when  \`Header\` is mounted [\`Header\`] 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <figure
      class="header__container-img"
    >
      <a
        href="/"
      >
        <img
          alt="logo"
          class="header__container-img--img"
          src="logoTipo.png"
        />
      </a>
    </figure>
    <nav
      class="header__nav"
    >
      <a
        href="/login"
      >
        <button
          class="header__nav--login"
          type="button"
        >
          Login
        </button>
      </a>
      <a
        href="/sign-up"
      >
        <button
          class="header__nav--sign-up"
          type="button"
        >
          Sign Up
        </button>
      </a>
    </nav>
  </header>
</DocumentFragment>
`;
